cime/scripts/lib/CIME/SystemTests/system_tests_common.py

```diff
@@ -474,7 +474,7 @@ def _compare_throughput(self):
                         diff = (baseline - current)/baseline
                         tolerance = self._case.get_value("TEST_TPUT_TOLERANCE")
                         if tolerance is None:
-                            tolerance = 0.25
+                            tolerance = 0.1
```

Comment:
@amametjanov , I just noticed this change during my latest CIME split. Can you explain why this change was needed? It's an impactful change since it changes the default fallback tput tolerance (when no TEST_TPUT_TOLERANCE is defined in config_machines.xml) on every machine for both E3SM and CESM.

Reply:
Throughput tolerance was both too wide (25%) and ignored (not a real regression test fail). For tests with baselines, it should be useful to know about greater than 10% variance (even if TEST_TPUT_TOLERANCE is not defined for the machine). This is what happened on Compy without any notice.

Reply:
@amametjanov , thanks